src/Common/Volnitsky.h

```diff
@@ -729,15 +727,15 @@ class MultiVolnitskyBase
 
 
 using Volnitsky = VolnitskyBase<true, true, ASCIICaseSensitiveStringSearcher>;
-using VolnitskyUTF8 = VolnitskyBase<true, false, ASCIICaseSensitiveStringSearcher>; /// exactly same as Volnitsky
+using VolnitskyUTF8 = VolnitskyBase<true, false, UTF8CaseSensitiveStringSearcher>;
```

Comment:
Why VolnitskyUTF8 used to use ASCII searcher? Doesn't look like type, because there was a comment